pipeline/console/__init__.py

```diff
@@ -0,0 +1,71 @@
+import argparse
+import sys
+
+import requests
+
+from pipeline import configuration
+from pipeline.util.logging import _print
+
+
+def main(args) -> int:
```

Comment:
I guess this is fine for now for a very simple CLI but might want to look at using a tool like [Typer](https://typer.tiangolo.com/) or [Click](https://click.palletsprojects.com/en/8.1.x/) to make it easier to work with as it grows